### ISSUE TITLE
Many tokens are checked to be linked to actor data when they shouldn't be

### ISSUE THREAD

**Issue by etriebe** [kakaroto/R20Converter#73]
Currently there are many token types that will be linked to their actor data. Like with Goblins in the hideout in the first chapter of Lost Mines of Phandelver. 

My expectation would be that if a character is in the Named NPCs folder structure they'd be linked. Else they'd be unlinked.

**Comment by kakaroto** [comment]
NPCs are by default unlinked, and PCs are linked. This is the second time someone is reporting this for the goblin so I'm wondering if Roll20 messed up (as usual) and made the Goblin a PC instead of an NPC. Can you verify that on your side in the Roll20 campaign ?
Thanks.

**Comment by etriebe** [comment]
It is an NPC in Roll20.

**Comment by jeremyeng** [comment]
I was running into this issue as well, I wrote a quick script macro to unlink all NPC tokens that aren't in the "Named NPCs" folder:

```
for (let token of canvas.tokens.placeables) {
  if (token.actor.data.type === "npc" && game.folders.get(token.actor.data.folder).data.name !== "Named NPCs") {
    token.update({actorLink: false}).then((token) => {
      ui.notifications.info(`${token.data.name} unlinked`);
    });
  }
}
```

**Comment by etriebe** [comment]
I did the same but for an offline script that did that recursively, since I
*think* some folders might be inside "Named NPCs".

**Comment by kakaroto** [comment]
This was fixed in the v0.12.2 release from May 2nd.
